modules/DoubleClick/resources/mw.DoubleClick.js

```diff
@@ -1697,6 +1710,16 @@
         },
         // Handler for various ad errors.
         onAdError: function( errorEvent ) {
+            if (this.nonFatalError) {
```

Comment:
Maybe better to create a `onNonFatalAdError` method, as the handling is quite unique for fatal and non fatal errors.